### ISSUE TITLE
render markdown for narrative text in swagger files

### ISSUE THREAD

**Issue by kbhawkey** [russell/fairy-slipper#52]
As an alternative to rendering narrative text formatted as rst, directly output descriptions as Markdown.

**Comment by annegentle** [comment]
Yeah I think swagger files themselves need markdown in the json objects.
